app/src/main/java/app/wefridge/wefridge/WeFridgeFirebaseMessagingService.kt

```diff
@@ -0,0 +1,92 @@
+package app.wefridge.wefridge
+
+
+import android.annotation.SuppressLint
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.util.Log
+import android.widget.RemoteViews
+import androidx.core.app.NotificationCompat
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+
+const val channelId="notification_channel"
+const val channelName="app.wefridge.wefridge"
+
+class WeFridgeFirebaseMessagingService : FirebaseMessagingService() {
+    /**
+     * Called when message is received.
+     *
+     * @param remoteMessage Object representing the message received from Firebase Cloud Messaging.
+     * It will check if a message contains a payload, if it does, the information will be used in
+     * the function generateNotification.
+     */
+    override fun onMessageReceived(remoteMessage: RemoteMessage) {
+        super.onMessageReceived(remoteMessage)
+
+        if (remoteMessage.notification !=null) {
+            Log.d("FCMService", "Message data payload: ${remoteMessage.data}")
+            generateNotification(remoteMessage.notification!!.title!!,remoteMessage.notification!!.body!!)
+        }
+    }
+
+    /**
+     * Called the FCM registration token at the beginning of the installation.
```

Comment:
I don't understand this comment. Does it mean "called **by** the FCM registration token", or "**Call** the FCM registration" or something else